scripts/build/esmodule-builder.js

```diff
@@ -0,0 +1,13 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'production';
```

Comment:
Is there a reason we default this to production? (I'm not sure whether it's ever not defined or what the default behavior is otherwise.)

It seems like for the most part this script could probably be replaced with a single shell command? Do we even need this script?

Reply:
I see what you're saying, since we're only ever going to run this for 'prod'. Also I can't find any documentation for `tsc` running differently with a different NODE_ENV, nor does it matter when I manually change it from the command line.

I will inline this in package.json